daemon/SecretsImpl/pluginfunctionwrappers_p.h

```diff
@@ -99,6 +99,16 @@ struct FoundResult {
     Sailfish::Secrets::Result result;
 };
 
+struct FoundLockedResult {
+    FoundLockedResult(bool f = false, bool l = false, const Sailfish::Secrets::Result &r = Sailfish::Secrets::Result())
```

Comment:
bool found = false, bool locked = false, const Sailfish::Secrets::Result &result = Sailfish::Secrets::Result() should not cause issues here with the struct members.

Reply:
yes, but internal struct, and it's fairly self documenting, will leave like this for consistency with the rest of concurrent function wrapper result structs